scripts/ingest-tarball.sh

```diff
@@ -258,11 +258,14 @@ version=$(echo "${tar_file_basename}" | cut -d- -f2)
 contents_type_dir=$(echo "${tar_file_basename}" | cut -d- -f3)
 tar_first_file=$(tar tf "${tar_file}" | head -n 1)
 tar_top_level_dir=$(echo "${tar_first_file}" | cut -d/ -f1)
-# Handle longer prefix with project name in dev.eessi.io
+# Handle longer prefix with project name in dev.eessi.io and
+# get 
```

Comment:
and get... the right basedir? :smile:

Reply:
:facepalm: :sweat_smile: , I didn't push the latest commit

Reply:
Fixed in badfbf8a52768efd7959252ae7eb3d4e84a81231